docs-v2/reference/cli.mdx

```diff
@@ -56,7 +56,7 @@ nango [command] --help
 
 | Command                        | Description                                                                                                                                       | Details                                                                                              |
 | -                              | -                                                                                                                                                 | -                                                                                                    |
-| `nango init`                   | Creates the `nango-integrations` directory with a demo Github integration.                                                                        | Generates `models.ts`, but not the compiled `.js` files.                                             |
+| `nango init`                   | Creates an integrations directory with a demo Github integration.                                                                        | Generates `models.ts`, but not the compiled `.js` files.                                             |
```

Comment:
`nango init` without additional param doesn't create any directory. Should it be `nango init DIR | Creates an DIR  integrations directory....`

Reply:
Good call. Updated in ac94019b7.